compiler_opt/rl/compilation_runner.py

```diff
@@ -160,6 +147,9 @@ def __init__(self):
     self._done = False
     self._lock = threading.Lock()
 
+  def enable(self):
+    self._done = False
```

Comment:
we don't _need_ it, because we wait for all shutdown futures to be fulfilled first, but it definitely doesn't hurt to put it there for consistency (and potential later maintenance issues).